src/dnm/bcf2Qcall.cc

```diff
@@ -203,7 +204,7 @@ int bcf_2qcall(const bcf_hdr_t *hdr, bcf1_t *rec, Trio t, qcall_t *mom_snp,
         for(j = 0; j < pl_fields[i].size() && pl_fields[i][j]; j++);
 
         // Estimate the depth using I16 fields 1 to 4, divided by num of samples
-        int d = (int)((double)d_rest / (n_samples - i) + .4999);
+        int d = (int)((double)d_rest / (/*n_samples*/3 - i) + .4999);
```

Comment:
Should a `static_cast<int>(static_cast<double>(...) + 0.4999)` be used here instead of C-style casts?